lib/bamboo/email.ex

```diff
@@ -117,7 +117,12 @@ defmodule Bamboo.Email do
 
   for function_name <- @address_functions do
     @doc """
-    Sets the #{function_name} on the email
+    Sets the `#{function_name}` on the email
+
+    ```
+    new_email
+    |> #{function_name}(["sally@example.com", "james@example.com"])
```

Comment:
Almost everywhere else we are

https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/adapter.ex#L15
https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/formatter.ex#L39
https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/phoenix.ex#L12
https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/test.ex#L31
https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/plug/email_preview_plug.ex#L13
https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/adapters/mailgun_adapter.ex#L10

Thanks for pointing out Bamboo.Email though. Those should be changed to 4 spaces of indentation. Oops!